python-backend/app/api/parties/party/models/address.py

```diff
@@ -0,0 +1,64 @@
+from sqlalchemy.dialects.postgresql import UUID
+from sqlalchemy.schema import FetchedValue
+from sqlalchemy.orm import validates
+
+from ....utils.models_mixins import Base
+from app.extensions import db
+
+class Address(Base):
+    __tablename__ = "address"
+    address_id = db.Column(db.Integer, primary_key=True)
+    suite_no = db.Column(db.String, nullable=True)
+    address_line_1 = db.Column(db.String, nullable=True)
+    address_line_2 = db.Column(db.String, nullable=True)
+    city = db.Column(db.String, nullable=True)
+    sub_division_code = db.Column(db.String,
+                                  # FIXME: Workaround for code-first tests
```

Comment:
It might not be needed now with James' test changes. At the time of writing, it is because the test DB is generated using SQLAlchemy models (instead of our migration files). The teardown kept blowing up because of the foreign key constraint between addresses and the `sub_division_code` table. To fix this, I instructed the teardown to just cascade the delete to all of its dependents (instead of blowing up).

Reply:
neat